ConfigurationSystem/Client/CSAPI.py

```diff
@@ -411,12 +411,12 @@ def addShifter( self, shifters = None ):
     :return: S_OK/S_ERROR
     """
 
-    def getOpsSection( ):
+    def getOpsSection():
```

Comment:
This is a method which is raising Exception contrary to DIRAC conventions. Should return S_ERROR() instead